SimPEG/directives/directives.py

```diff
@@ -2300,24 +2455,47 @@ def update(self):
             else:
                 jtj_diag += sim.getJtJdiag(m, W=dmisfit.W)
 
-        # Normalize and threshold weights
+        # Compute and sum root-mean squared sensitivities for all objective functions
         wr = np.zeros_like(self.invProb.model)
         for reg in self.reg.objfcts:
             if not isinstance(reg, BaseSimilarityMeasure):
                 wr += reg.mapping.deriv(self.invProb.model).T * (
                     (reg.mapping * jtj_diag) / reg.regularization_mesh.vol ** 2.0
                 )
-        if self.normalization:
-            wr /= wr.max()
-        wr += self.threshold
+
         wr **= 0.5
+
+        # Apply thresholding
+        if self.threshold_method == "global":
```

Comment:
Just making a note here that this slightly changes the behavior from before, with the default now being the `global` method meaning the threshold is applied before normalization. Does the `percentile` or `amplitude` threshold option do anything similar to this? (I feel like the `amplitude` threshold comes closest but it still clips and doesn't add.) Not saying this is a blocking change, just wanted to make note and allow others to comment if they wanted to.

Reply:
Im not worried